src/main/java/com/rtb/projectmanagementtool/task/TaskController.java

```diff
@@ -54,6 +126,21 @@ public TaskData getTaskByID(long taskID) {
     return task;
   }
 
+  public ArrayList<TaskData> getTasksByUserID(long userID) {
+    Filter filter = new FilterPredicate("users", FilterOperator.EQUAL, userID);
+    return getTasks(filter, NO_QUERY_LIMIT, NO_QUERY_SORT);
+  }
+
+  public ArrayList<TaskData> getTasksByProjectID(long projectID) {
+    Filter filter =
+        new CompositeFilter(
+            CompositeFilterOperator.AND,
+            Arrays.asList(
+                FilterOperator.EQUAL.of("projectID", projectID),
+                FilterOperator.EQUAL.of("parentTaskID", 0)));
```

Comment:
This wouldn't return subtasks I believe?  This should be ok, just make sure to document somewhere (comment here or design doc) and clarify with the rest of the team that the project page will only display the first level tasks.

Reply:
Yeah I thought returning all subtask layers would be more complicated and wouldn't be useful for most scenarios where someone would just want to see the general idea of a project. I was also thinking that maybe after mvp, an extra feature can be added where a user can choose the amount of layers of nested tasks to display. I can add this to the design doc.